ironic-inspector-scripts/runhttpd-inspector

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/bash
```

Comment:
One of the key points of this work is to not run two httpd instances, only one to Rule Them ALL!

Reply:
yes, I changed the name here because when we run httpd for ironic, we can run runhttpd normally, while when running inspector we run runhttpd-inspector, configurations are quite different
I can have a look at merging the two scripts in one, but in any case we won't end up running two instances, there will be only one depending on which service we're running

Reply:
still need to change this